scripts/retrieve_databundle_light.py

```diff
@@ -354,6 +356,73 @@ def download_and_unzip_direct(config, rootpath, hot_run=True, disable_progress=F
     return True
 
 
+def download_and_unzip_hydrobasins(
+    config, rootpath, hot_run=True, disable_progress=False
+):
+    """
+    download_and_unzip_basins(config, rootpath, dest_path, hot_run=True,
+    disable_progress=False)
+
+    Function to download and unzip the data for hydrobasins.
+
+    Inputs
+    ------
+    config : Dict
+        Configuration data for the category to download
+    rootpath : str
+        Absolute path of the repository
+    hot_run : Bool (default True)
+        When true the data are downloaded
+        When false, the workflow is run without downloading and unzipping
+    disable_progress : Bool (default False)
+        When true the progress bar to download data is disabled
+
+    Outputs
+    -------
+    True when download is successful, False otherwise
+    """
+    resource = config["category"]
+    url_templ = config["urls"]["hydrobasins"]
+    suffix_list = config["urls"]["suffixes"]
+
+    basins_fl = snakemake.config["renewable"]["hydro"]["resource"]["hydrobasins"]
+    level_pattern = r".*?lev(.*)_.*"
+    level_code = re.findall(level_pattern, basins_fl)
+
+    for rg in suffix_list:
+        # lev01_v1c.zip
+        url = url_templ + "hybas_" + rg + "_lev" + level_code + "_v1c.zip"
+        file_path = os.path.join(config["destination"], os.path.basename(url))
+        if hot_run:
+            if os.path.exists(file_path):
+                os.remove(file_path)
+
+            try:
+                logger.info(f"Downloading resource '{resource}' from cloud '{url}'.")
+                progress_retrieve(
+                    url,
+                    file_path,
+                    headers=[("User-agent", "Mozilla/5.0")],
+                    disable_progress=disable_progress,
+                )
+
+                # if the file is a zipfile and unzip is enabled
+                # then unzip it and remove the original file
+                if config.get("unzip", False):
```

Comment:
It is always a zip file in the hydrobasins, right?
If so, maybe this if is not needed

Reply:
Fixed :)